process-automation/latest/model/connectors/docusign.md

```diff
@@ -2,103 +2,214 @@
 
-The input parameters to request a DocuSign signature are:
+As part of the BPMN definition process, any service task that is responsible for sending the document needs to set the `docusignconnector.SEND_FOR_SIGNATURE` or `docusignconnector.DOWNLOAD_DOCUMENT` properties as the value for its implementation attribute.
+
+The following input parameters must be provided for DocuSign API in the Service task depending on the implementation:
+
+The input parameters for `SEND_FOR_SIGNATURE` are:
 
 | Parameter | Type | Description |
 | --------- | ---- | ----------- |
 | file | File | *Required.* A [variable]({% link process-automation/latest/model/processes/index.md %}#process-variables) of type file containing the document to be signed. |
 | documentId | Integer | *Optional.* Correlation ID to send to the DocuSign API, for example `250`. |
 | nodeFormat | String | *Optional.* The format of the document to be signed. Values are `pdf` or `docx`. |
 | outputFileName | String | *Optional.* The name of the file that will be created, for example `invoice.pdf`. |
-| recipientEmail | String | *Required.* The email address of the person signing the document. |
-| recipientName | String | *Optional.* The name of the person signing the document, for example `John Doe`. |
+| recipientEmail | String | *Required.* The email address of the signer when the document is going to be signed by only one signer. **Note:** The connector can only receive either **signers** or **recipientEmail** but not both at the same time.  |
+| recipientName | String | *Optional.* The name of the signer i.e. the email recipient, when the document is going to be signed by only one signer. |
 | emailSubject | String | *Optional.* The subject line of the email sent with the document to sign. |
-| signHerePage | String | *Optional.* The page number in the document the `Sign Here` box will appear on, for example `3`. |
-| signHereX | String | *Optional.* The X position of the `Sign Here` box in the document, for example `100`. |
-| signHereY | String | *Optional.* The Y position of the `Sign Here` box in the document, for example `50`. |
-| targetFileMetadata | Content-Metadata | *Optional.* Metadata to store the file with. This is a JSON object of key value pairs. See below for an example. |
+| signHerePage | String | *Optional.* The page number in the document the `Sign Here` box will appear on, when the document is going to be signed by only one signer, for example `3`. |
+| signHereX | String | *Optional.* The X position of the `Sign Here` box in the document, when the document is going to be signed by only one signer, for example `100`. |
+| signHereY | String | *Optional.* The Y position of the `Sign Here` box in the document, when the document is going to be signed by only one signer for example `50`. |
+| targetFileMetadata | Content-Metadata | *Optional.* Metadata to store the file with. This is a JSON object of key value pairs. |
```

Comment:
targetFileMetadata is not needed for this action.

Reply:
Removed.